src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -6743,6 +6743,74 @@ public final class TokenTypes {
     public static final int EMBEDDED_EXPRESSION_END =
             JavaLanguageLexer.EMBEDDED_EXPRESSION_END;
 
+    /**
+     * The {@code _} reserved keyword. Appears as part of a variable definition in
+     * place of an identifier value, and is not used as a token type explicitly. This
+     * is to maintain backwards compatibility with older versions of Java that considered
+     * {@code _} to be a valid identifier. There is no reason to use this token type
+     * directly.
```

Comment:
This makes me question why we are even adding this token to the public list and just don't want to hide this token away from the world. It looks like we already hide it from the AST display (please onfirm), so why even show this?

Reply:
Good point, I will remove this :)